### DISCUSSION TITLE
Centre Selection example

### DISCUSSION THREAD

**Discussion by 0x6900** [sushrut111/cowin-automation-extn#28] [Q&A] [answered]
How to choose centre preference ? Any example of it ?

**Comment by sushrut111** [comment, marked as answer]
Take an example of these centers

![image](https://user-images.githubusercontent.com/11269256/119502784-eed36c80-bd87-11eb-95f9-724e067912c9.png)

I would try find unique words from these. Also I am fine with anything in pincode 411037

So I would set 

![image](https://user-images.githubusercontent.com/11269256/119503120-3fe36080-bd88-11eb-965d-fc39228bb40b.png)


Also know that, 
- the comma separated strings are considered as separate preferences and they are trimmed and converted to lowercase 
- address of each center = lowercase ( name of center + address + pincode)
- now these center preferences are checked against the address to see if any of them exists as the address' substring.
- if yes, the first match is selected to be booked if slots > [min availability specified by user]
